package.json

```diff
@@ -18,6 +18,7 @@
 		"coverage": "nyc report --reporter=text-lcov | coveralls",
 		"integrationtests": "ts-mocha --require test/testhook.js \"test/specs/parsing/**/*.ts\" -p test/tsconfig.json",
 		"prepare": "npm run build",
+		"performance": "ts-node --project test/tsconfig.json -r tsconfig-paths/register test/qt3testsBenchmark.ts",
```

Comment:
Can we also use `-p` instead of `--project` here for consistency.

Reply:
-P is used